allennlp/modules/transformer/self_attention.py

```diff
@@ -116,7 +117,7 @@ def forward(
             mask_reshp = (batch_size, 1, 1, k_length)
             attention_mask = (attention_mask == 0).view(mask_reshp).expand_as(
                 attention_scores
-            ) * -10e5
+            ) * nn_util.min_value_of_dtype(attention_scores.dtype)
```

Comment:
Is that really what we use? Let's use whatever huggingface uses. `min_value` seems risky, because of numeric weirdnesses at the edge of floating point precision.

Reply:
The old value is a bit weird too though. `-10e5`? Why wouldn't you write `-1e6` then? 🤷

Reply:
https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_utils.py#L272